tests/cross/src/test/scala/tests/pc/CompletionSnippetSuite.scala

```diff
@@ -52,6 +52,21 @@ class CompletionSnippetSuite extends BaseCompletionSuite {
        |""".stripMargin
   )
 
+  checkSnippet(
+    "nullary2",
```

Comment:
I think this case is more like `nilary`, instead of `nullary`
https://stackoverflow.com/questions/7600910/difference-between-function-with-parentheses-and-without

Reply:
Ok cool! Changed the name